canvas_modules/common-canvas/__tests__/palette/palette-flyout-test.js

```diff
@@ -162,8 +162,8 @@ describe("Palette renders correctly", () => {
 		const wrapper = createMountedPalette();
 		const importCat = findCategoryElement(wrapper, "Import");
 		importCat.simulate("click");
-		expect(wrapper.find(PaletteFlyoutContentList)).to.have.length(1);
-		expect(wrapper.find(PaletteContentListItem)).to.have.length(3);
+		expect(wrapper.find(PaletteFlyoutContentList)).to.have.length(2);
+		expect(wrapper.find(PaletteContentListItem)).to.have.length(5);
```

Comment:
The code in this test which looks for PaletteFlyoutContentList and PaletteContentListItem are not testing if the categories are open or closed.  Can we use the code from the  test below this one 
```
	const categoryList = wrapper.find("div.palette-flyout-categories");
	expect(categoryList.find(".bx--accordion__item--active")).to.have.length(2);
```
to check how many categories are open? 

Perhaps you could add these two lines to a separate function and then call it from all the places where we want to check how many categories are open.

Reply:
categoriesOpenCounts function has been created !!